app-admin/bitwarden-desktop-bin/bitwarden-desktop-bin-1.28.3.ebuild

```diff
@@ -67,11 +66,11 @@ src_install() {
 	fperms 755 /opt/Bitwarden/bitwarden
 	fperms 4755 /opt/Bitwarden/chrome-sandbox
 
-	insinto /usr/share/applications
-	doins usr/share/applications/bitwarden.desktop
+	domenu usr/share/applications/bitwarden.desktop
 
-	insinto /usr/share/icons
-	doins -r usr/share/icons/hicolor
+	for x in 16 32 48 64 256 512
+		do doicon -s ${x} data/icons/hicolor/${x}*/*
+	done
```

Comment:
That's uncommon style. Please do the usual `for x in ...; do`

Reply:
Indeed, fully agree :-)